### PR TITLE
Update OTLP metrics exporter Go module name

### PR DESCRIPTION
**Pull request by cwaldren-ld**
The module name is incorrect, leading `go get` to fail.

### CODE REVIEW

**Review by dashpole — APPROVED** [review]

**Comment by cwaldren-ld** [comment]
I'm blocked on getting the CLA signoff for my company. If anyone wants to just duplicate this and close this PR, that'd be fine, otherwise it might take a couple days.

**Review by MadVikingGod — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

**Comment by MrAlias** [comment]
Closing in favor of https://github.com/open-telemetry/opentelemetry-go/pull/3259

**Pull request closed without merging**
